domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1006,6 +1008,23 @@ public void optimizedSqlCanSetSize() {
     assertThat(PgUtil.getOptimizedSqlSize(), is(oldSize));
   }
 
+  @Test
+  public void getTotalRecordsTest() {
+    assertNull(PgUtil.getTotalRecords(10, null, 0, 0));
+
+    assertEquals(PgUtil.getTotalRecords(10, 20, 0, 0),  (Integer)20);
+
+    assertEquals(PgUtil.getTotalRecords(0, 20, 10, 10),  (Integer)10);
+
+    assertEquals(PgUtil.getTotalRecords(10, 30, 10, 10),  (Integer)30);
+
+    assertEquals(PgUtil.getTotalRecords(10, 20, 20, 10),  (Integer)30);
+
+    assertEquals(PgUtil.getTotalRecords(5, 20, 20, 10), (Integer) 25);
+
+    assertEquals(PgUtil.getTotalRecords(5, 20, 20, 10), (Integer) 25);
```

Comment:
Either use

    assertEquals(expected, actual);

or

    assertThat(actual, is(expected));

Note the different order of expected and actual. Otherwise we get misleading error messages when an assertion fails.

Reply:
Thanks, fixed.